src/log_utils.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Tlf - contest logging program for amateur radio operators
+ * Copyright (C) 2019 Thomas Beierlein <dl1jbe@darc.de>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+/* ------------------------------------------------------------------------
+*    util functions to work with lines from log
+*
+---------------------------------------------------------------------------*/
+#include <string.h>
+#include <stdbool.h>
+
+/** check if logline is only a comment */
+bool log_is_comment(char *buffer) {
+
+    if (buffer[0] != ';' && strlen(buffer) > 60) /** \todo better check */
```

Comment:
what could be the reason for the ">60"?  it was in the code earlier, just don't get the idea behind it.

Reply:
> what could be the reason for the ">60"? it was in the code earlier, just don't get the idea behind it.

Good question. I assume in earlier versions there were problems with short log lines sometimes. Had a look into old versions: It was already there in 0.9.10.

Until now I modified code very conservative - especially if it was not clear what the code was for.

My plan was to isolate all the small manipulations of logline into one file, write tests for it and afterwards clean up.

Reply:
OK.  :+1: 
There is still a lot to improve, step by step.